build-constraints.yaml

```diff
@@ -3603,6 +3603,417 @@ packages:
         - consul-haskell
         - hasql-transaction
 
+    # Prior to the Pantry rewrite of the curator tool, dependencies of
+    # packages added to Stackage were automatically added. With the
+    # Pantry rewrite, this is no longer the case. The following
+    # packages were "grandfathered in" during this
+    # transition. Ideally, over time, they'll be removed from here and
+    # added to specific maintainers.
+    "Grandfathered dependencies":
+        - random
+        - QuickCheck
+        - quickcheck-io
+        - fingertree-psqueue
+        - test-framework
+        - test-framework-hunit
+        - test-framework-quickcheck2
+        - Glob
+        - data-binary-ieee754
+        - data-memocombinators
+        - temporary
+        - attoparsec
+        - cereal
+        - ChasingBottoms
+        - Decimal
+        - Diff
+        - EdisonCore
+        - HDBC
+        - HDBC-session
+        - HTTP
+        - HsOpenSSL
+        - ListLike
+        - MemoTrie
+        - Only
+        - PSQueue
+        - ParsecTools
+        - RSA
+        - SegmentTree
+        - X11
+        - alsa-mixer
+        - ansi-terminal
+        - appar
+        - asn1-encoding
+        - asn1-parse
+        - asn1-types
+        - authenticate
+        - auto-update
+        - base-compat
+        - base-orphans
```

Comment:
I'm a bit confused here—aren't I already listed as the maintainer for `base-orphans`? Similarly, someone already maintains `base-compat`.

Reply:
It's entirely possible that there are mistakes in this PR, it was a quick first attempt to test out the concept with a new curator tool.